csvPlotter/reader.py

```diff
@@ -29,16 +29,16 @@ def computeWeekViolations(self, data, results):
             results["eff_violations"] = results["violations"]
 
     def process(self):
-        self.computeWeekViolations(self.data.head(self.week_length), self.week1)
-        self.computeWeekViolations(self.data.tail(self.week_length), self.week2)
+        self.computeWeekViolations(self.data.head(self.week_length), self.week)
+        #self.computeWeekViolations(self.data.tail(self.week_length), self.week2)
         self.total_pages = self.data.sum()
-        self.mean_pages = self.total_pages / (2 * self.week_length)
-        self.violations = self.week1["violations"] + self.week2["violations"]
-        self.eff_violations = self.week1["eff_violations"] + self.week2["eff_violations"]
+        self.mean_pages = self.total_pages / self.week_length
+        self.violations = self.week["violations"] #+ self.week2["violations"]
+        self.eff_violations = self.week["eff_violations"] #+ self.week2["eff_violations"]
         # Weekly violation counting system skips the case where someone didn't read on the last day of week 1 and first
         # day of week 2
-        if self.data.iloc[self.week_length - 1] < self.daily_target and \
-           self.data.iloc[self.week_length] < self.daily_target:
+        if self.data.iloc[self.week_length - 1] < self.daily_target: #and \
+           #self.data.iloc[self.week_length] < self.daily_target:
```

Comment:
This would result in one extra violation being recorded for a given week. We should remove this entirely for week-long sprints.